frontend/cypress/e2e/patientEntry.cy.js

```diff
@@ -36,12 +41,21 @@ describe("Patient Search", function () {
       );
     });
   });
+
   it("User should click save new patient information button", function () {
     patientPage.clickSavePatientButton();
     cy.wait(1000);
     cy.get("div[role='status']").should("be.visible");
     cy.wait(200).reload();
   });
+  it("Should search patient By Date of Birth", function () {
+    cy.wait(1000);
```

Comment:
Well done @manishjha-04 , i see a redundant/ duplication of code, especially on line 52, Can you write/refactor out this line and simply call that function where necessary.

Reply:
Hey @sherrif10, could you please provide more clarification on what you are talking about?